src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea, Colorable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public double getArea() {
+        return 0;
+    }
```

Comment:
The `getArea` method is a placeholder here and should be implemented in the concrete figure classes. Each figure class should override this method to provide the correct area calculation based on the figure's properties.